.infrastructure/98_variables.tf

```diff
@@ -24,4 +24,19 @@ variable "tags" {
 ### required for security group ALB, ECS and RDS
 variable "cms_app_port" {
   default = 1337
+}
+
+variable "fargate_cpu" {
+  description = "Fargate instance CPU units to provision (1 vCPU = 1024 CPU units)"
+  default     = "4096" ##### 4 vCPU
+}
+
+variable "fargate_memory" {
+  description = "Fargate instance memory to provision (in MiB)"
+  default     = "16384" ##### 16 GB RAM
```

Comment:
```suggestion
  default     = "3072"
```

Reply:
ok, I'll make the change with the next commit